src/main/java/org/osiam/ApplicationInitializer.java

```diff
@@ -0,0 +1,27 @@
+package org.osiam;
+
+import javax.servlet.ServletContext;
+import javax.servlet.ServletException;
+import javax.servlet.ServletRegistration;
+
+import org.springframework.web.WebApplicationInitializer;
+import org.springframework.web.context.ContextLoaderListener;
+import org.springframework.web.context.support.AnnotationConfigWebApplicationContext;
+import org.springframework.web.servlet.DispatcherServlet;
+
+public class ApplicationInitializer implements WebApplicationInitializer {
+
+    @Override
+    public void onStartup(ServletContext servletContext) throws ServletException {
```

Comment:
Yes, added in [line 28](https://github.com/osiam/resource-server/pull/50/files#diff-b4a0030bff71a7d7121999044d4de3f4R28)